src/main/java/labs/introtoprogramming/lab5/scene/Camera.java

```diff
@@ -0,0 +1,9 @@
+package labs.introtoprogramming.lab5.scene;
+
+import labs.introtoprogramming.lab5.geometry.Vector3;
+
+public interface Camera {
```

Comment:
Maybe make it an abstract class or a class that extends SceneObject? I think it is logical to treat camera as part of scene and manipulate it.

Reply:
My initial assumption was that we will need objects on scene to be divided into three categories:
- regular objects, which are rendered
- camera (s)
- lights, which are accessed when computing shadows.
But since all of them share some common properties, it is a good idea indeed to make them extend the same class.